src/index.html

```diff
@@ -7,7 +8,58 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion-->
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+
+      <div class="box stars--0">
```

Comment:
as I see in requirements, this block should be `stars`, not `box`
```suggestion
      <div class="stars stars--0">
```